test/readline/test_readline.rb

```diff
@@ -496,7 +496,8 @@ def test_interrupt_in_other_thread
       begin
         Thread.new{
           trap(:INT) {
-            p :INT
+            puts 'TRAP'
+            $stdout.flush
```

Comment:
@aycabta I wonder if there is some problem here. I noticed that `#flush` was necessary to make it pass locally. I wonder if this is part of the problem we are seeing on Readline 7.0.